graphql_server/__init__.py

```diff
@@ -85,22 +86,15 @@ def for_site(self, site: Site) -> 'SPlans':
         )
 
 
-class PlanManager:
+class PlanManager(metaclass=Singleton):
```

Comment:
Do we really need metaclasses? Can't we perhaps do something simpler?

https://realpython.com/python-metaclasses

Reply:
A Base class can get the same results but metaclass is the recommended singleton implementation: https://stackoverflow.com/questions/6760685/creating-a-singleton-in-python

Reply:
Okay, sounds good.